[libprelude] Unable to install

```
ImportError: No module named six
make[4]: *** [Makefile:1887: install-data-local] Error 1
make[4]: Leaving directory '/usr/local/tmp/crew/libprelude-4.1.0.tar.gz.dir/libprelude-4.1.0/docs/api'
make[3]: *** [Makefile:1632: install-am] Error 2
make[3]: Leaving directory '/usr/local/tmp/crew/libprelude-4.1.0.tar.gz.dir/libprelude-4.1.0/docs/api'
make[2]: *** [Makefile:1471: install-recursive] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/libprelude-4.1.0.tar.gz.dir/libprelude-4.1.0/docs'
make[1]: *** [Makefile:1649: install-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/libprelude-4.1.0.tar.gz.dir/libprelude-4.1.0'
make: *** [Makefile:1942: install] Error 2
libprelude failed to build: `make DESTDIR=/usr/local/tmp/crew/dest install` exited with 2
```

--- a/packages/ansifilter.rb
+++ b/packages/ansifilter.rb
@@ -8,8 +8,16 @@ class Ansifilter < Package
   source_sha256 '23d2cf439d4ed4fbec8050b2826d61c244694ce06aaf8ca7d0ec1016afebee3f'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ansifilter-2.10-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ansifilter-2.10-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ansifilter-2.10-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ansifilter-2.10-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'd5a8d33520c76aeb3f13067f169300d4b959a78ef352331b14a1b70db9f44f5f',
+     armv7l: 'd5a8d33520c76aeb3f13067f169300d4b959a78ef352331b14a1b70db9f44f5f',
+       i686: 'b312b46ae42dfb60620e6f62caa5aae83bfe259e01094d84f2dd949a137eedf2',
+     x86_64: 'af4c1a4f1b9fddd1efa5dabded1664880bd25fc17b52290af956d4d5d7dedf5c',
   })
 
   def self.build

--- a/packages/bash_completion.rb
+++ b/packages/bash_completion.rb
@@ -8,17 +8,22 @@ class Bash_completion < Package
   source_sha256 'c3f1bacac64bc69e3dfb9b1ef79ddbf755aac441b452845531950f1643bed25f'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/bash_completion-2.8-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/bash_completion-2.8-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/bash_completion-2.8-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/bash_completion-2.8-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'de8e10c9a5d3bf2be05d6fac39e10db2ab1d39c6c9d6b3f3acdec71b6d9f9f04',
+     armv7l: 'de8e10c9a5d3bf2be05d6fac39e10db2ab1d39c6c9d6b3f3acdec71b6d9f9f04',
+       i686: 'ee87f97c9a5ccec83d6f618fdc09c5af0ec93a095d717ff3fbc8b24137bf264a',
+     x86_64: '9efbe3b29ab29977df9c6f1cce34fb06677590bd2a4c44ad6da5b3d49b5f7d9c',
   })
 
-  depends_on 'autoconf'
-  depends_on 'automake'
-
   def self.build
-    system "autoreconf -i"
-    system "./configure"
-    system "make"
+    system 'autoreconf -i'
+    system "./configure --prefix=#{CREW_PREFIX}"
+    system 'make'
   end
 
   def self.install
@@ -29,10 +34,9 @@ class Bash_completion < Package
     puts
     puts "To complete installation, execute the following:".lightblue
     puts "echo '# bash completion' >> ~/.bashrc".lightblue
-    puts "echo '[[ $PS1 && -f /usr/local/share/bash-completion/bash_completion ]] && \\' >> ~/.bashrc".lightblue
-    puts "echo '    . /usr/local/share/bash-completion/bash_completion' >> ~/.bashrc".lightblue
+    puts "echo '[[ $PS1 && -f #{CREW_PREFIX}/share/bash-completion/bash_completion ]] && \\' >> ~/.bashrc".lightblue
+    puts "echo '  source #{CREW_PREFIX}/share/bash-completion/bash_completion' >> ~/.bashrc".lightblue
     puts "source ~/.bashrc".lightblue
     puts
   end
-
 end

--- a/packages/dmxproto.rb
+++ b/packages/dmxproto.rb
@@ -8,10 +8,16 @@ class Dmxproto < Package
   source_sha256 'a911a086a61c1bb16d35f70b391f167744ee721b5e2a7f22c00bc5a2c1ecb242'
 
   binary_url ({
-
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/dmxproto-2.3-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/dmxproto-2.3-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/dmxproto-2.3-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/dmxproto-2.3-0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-
+    aarch64: 'a6e150081dd982846398d9d2fb4d83e5b254e3997d9bc02bd431e7e042426a85',
+     armv7l: 'a6e150081dd982846398d9d2fb4d83e5b254e3997d9bc02bd431e7e042426a85',
+       i686: '2f22e737feddb8e70a1093e708734ecae9750204680bee7e9226b0bc9b04d8e4',
+     x86_64: 'b47b4acf2dcc24009ccd1f9e3200d2ea0309682e13fc7fcf7d570ae36157f91d',
   })
 
   def self.build

--- a/packages/fontconfig.rb
+++ b/packages/fontconfig.rb
@@ -8,22 +8,28 @@ class Fontconfig < Package
   source_sha256 'a6ca290637d8b2c4e1dd40549b179202977593f7481ec83ddfb1765ad90037ba'
 
   binary_url ({
-
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/fontconfig-2.13.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/fontconfig-2.13.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/fontconfig-2.13.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/fontconfig-2.13.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-
+    aarch64: 'ced3440f18ee896c1623ecd1ef3c493c41f6947ee4c6b4c54cd6bd23080e42ff',
+     armv7l: 'ced3440f18ee896c1623ecd1ef3c493c41f6947ee4c6b4c54cd6bd23080e42ff',
+       i686: '64a8085ac2a3cb30c1d523133778a868ece074fa973c5f7b0548b9b639f7ecb2',
+     x86_64: '7df51056ad09bf3b74fc1e85e6049306322f55641b191f3a8fa0378281c71702',
   })
 
   depends_on 'expat'
   depends_on 'gperf'
-  depends_on 'pkgconfig'
   depends_on 'freetype'
+  depends_on 'util_linux'
 
   def self.build
     system "./configure \
-          --prefix=#{CREW_PREFIX} \
-          --libdir=#{CREW_LIB_PREFIX} \
-          --localstatedir=#{CREW_PREFIX}/cache"
+            --prefix=#{CREW_PREFIX} \
+            --libdir=#{CREW_LIB_PREFIX} \
+            --localstatedir=#{CREW_PREFIX}/cache"
     system 'make'
   end
 

--- a/packages/graphene.rb
+++ b/packages/graphene.rb
@@ -8,10 +8,16 @@ class Graphene < Package
   source_sha256 '410f2e848952cc5830f39b6f6ea7f9b0a487cfe99dad86eec6f22ccbb3ec635b'
 
   binary_url ({
-
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/graphene-1.8.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/graphene-1.8.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/graphene-1.8.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/graphene-1.8.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-
+    aarch64: '2a7f9e8dd9a28ef882bedbdd177824c4e51add7e7457f3908369277e7825890f',
+     armv7l: '2a7f9e8dd9a28ef882bedbdd177824c4e51add7e7457f3908369277e7825890f',
+       i686: 'dc070efb57d7b780e7d53993c9d531cca24a9b5cf96831b4904ffd5bc140a51a',
+     x86_64: '851d4dee78563bee5938d43bd263781ac3851fc43db93d9d946055ea44831b7b',
   })
 
   depends_on 'meson'
@@ -20,11 +26,10 @@ class Graphene < Package
   def self.build
     system "meson --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} -Darm_neon=false _build"           
     system "ninja -v -C _build"
-    system "ninja -C _build test"      
+    system "ninja -C _build test"
   end
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C _build install"
   end 
-  
 end

--- a/packages/keychain.rb
+++ b/packages/keychain.rb
@@ -8,8 +8,16 @@ class Keychain < Package
   source_sha256 'dcce703e5001211c8ebc0528f45b523f84d2bceeb240600795b4d80cb8475a0b'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/keychain-2.8.5-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/keychain-2.8.5-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/keychain-2.8.5-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/keychain-2.8.5-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'e494cfd41ed3215757cf656b36a7b483f8a0e007fb5963d2e6b058c701f6ff44',
+     armv7l: 'e494cfd41ed3215757cf656b36a7b483f8a0e007fb5963d2e6b058c701f6ff44',
+       i686: '539b7395d19f08d1a914070b889f240fe890fc0abf7b298d62597bcc72b9916f',
+     x86_64: '4b4aead7a52d97ef7acfc5695952e57b35c3c12935b4fde6ce3ed5e64705ec0a',
   })
 
   def self.install

--- a/packages/libprelude.rb
+++ b/packages/libprelude.rb
@@ -7,16 +7,31 @@ class Libprelude < Package
   source_url 'https://www.prelude-siem.org/attachments/download/831/libprelude-4.1.0.tar.gz'
   source_sha256 '21ee5bee2fc4136903c606ad7891927b7b32a29e92ca7e418ac458823951562d'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libprelude-4.1.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libprelude-4.1.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libprelude-4.1.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libprelude-4.1.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'cb7139c322d79adb0dfa2a8c87b63d0b496c6485afed3a10dd48fdfe93cdf6c6',
+     armv7l: 'cb7139c322d79adb0dfa2a8c87b63d0b496c6485afed3a10dd48fdfe93cdf6c6',
+       i686: '32ddcbd6646843c224450cb45a018873f89b4c1f66a4addef5dcf15318d9d953',
+     x86_64: 'f7642b1ffb9a0b5ee5c22a3f1345fbc0ff8b7e334a9839c7d24b7108fd069806',
+  })
+
   depends_on 'python27' => :build    
   depends_on 'python3' => :build
     
   def self.build
-      system "./configure \
-               --prefix=#{CREW_PREFIX} \
-               --libdir=#{CREW_LIB_PREFIX}"      
+    system "./configure \
+            --prefix=#{CREW_PREFIX} \
+            --libdir=#{CREW_LIB_PREFIX}"
   end
 
   def self.install
-      system "make DESTDIR=#{CREW_DEST_DIR} install"
+    system "pip install six"
+    system "make DESTDIR=#{CREW_DEST_DIR} install"
+    system "pip uninstall --yes six"
   end
 end

--- a/packages/libxcb.rb
+++ b/packages/libxcb.rb
@@ -8,10 +8,16 @@ class Libxcb < Package
   source_sha256 '092f147149d8a6410647a848378aaae749304d5b73e028ccb8306aa8a9e26f06'
 
   binary_url ({
-
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libxcb-1.12-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libxcb-1.12-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libxcb-1.12-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libxcb-1.12-0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-
+    aarch64: '2efffcb6d5442087cd196c813756491617097f4749f7ff0ce71f71d6b70ccbf0',
+     armv7l: '2efffcb6d5442087cd196c813756491617097f4749f7ff0ce71f71d6b70ccbf0',
+       i686: 'dc4c741c4b94c0cebe7568e561bd7a739038689dcbbfa558194e5cf365f6674b',
+     x86_64: 'ec938fc5753e9d7d5d67e38c4cf561be13a873d7b4343d4a9ca60d658dc35a70',
   })
 
   depends_on 'python27' => :build

--- a/packages/mesa.rb
+++ b/packages/mesa.rb
@@ -8,10 +8,16 @@ class Mesa < Package
   source_sha256 'e5915680d44ac9d05defdec529db7459ac9edd441c9845266eff2e2d3e57fbf8'
 
   binary_url ({
-
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-17.3.6-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-17.3.6-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-17.3.6-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-17.3.6-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-
+    aarch64: '3ea41fb9c2cb997a546e46d5f12211d8770a058c446ba423d66cd2c6b58be98d',
+     armv7l: '3ea41fb9c2cb997a546e46d5f12211d8770a058c446ba423d66cd2c6b58be98d',
+       i686: 'acd169040bb947b5e0cfbb03f9cb3a3a37b092f7fe5176513d196c4e633c6d09',
+     x86_64: 'b9b3aee03c279453044bb07bad1693501b81d29d66fe75f2cdf8e1a79b0754f9',
   })
 
   depends_on 'libdrm'
@@ -30,6 +36,7 @@ class Mesa < Package
 
   # tested on armv7l
   def self.build
+    system "pip install setuptools"
     system "pip install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR} Mako"
     system "./configure \
             --prefix=#{CREW_PREFIX} \
@@ -45,6 +52,7 @@ class Mesa < Package
             --enable-xa \
             --enable-dri"
     system "make"
+    system "pip uninstall --yes setuptools"
   end
 
   def self.install

--- a/packages/neofetch.rb
+++ b/packages/neofetch.rb
@@ -8,8 +8,16 @@ class Neofetch < Package
   source_sha256 '4808e76bd81da3602cb5be7e01dfed8223b1109e2792755dd0d54126014ee696'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/neofetch-3.3.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/neofetch-3.3.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/neofetch-3.3.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/neofetch-3.3.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'dd1d1bc23fcadf54c74a6800cc5a6824300f82d2965d0810079e5f589d3ae1cf',
+     armv7l: 'dd1d1bc23fcadf54c74a6800cc5a6824300f82d2965d0810079e5f589d3ae1cf',
+       i686: '7e0385951fd9a982eec5d3b6223ce83efa94501b4f0b74536ff55a129e2e33ea',
+     x86_64: 'f3ffcb5a6b7c3c5e5f663293db150fd6fde371b65067cc5dc3306a449e8eca75',
   })
 
   def self.build

--- a/packages/ondir.rb
+++ b/packages/ondir.rb
@@ -8,8 +8,16 @@ class Ondir < Package
   source_sha256 '475fe7bd447756b4effde87c8aa67e063d805b775785b9f39858422dc244aef2'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ondir-0.2.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ondir-0.2.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ondir-0.2.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ondir-0.2.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'd8b3ea69345a1a88511b5adf7005befa9c53a90c7df43e5a9be47ee4294faf6a',
+     armv7l: 'd8b3ea69345a1a88511b5adf7005befa9c53a90c7df43e5a9be47ee4294faf6a',
+       i686: '45b1160ad68b3942db66057e74a6b4ef2dce30fd1c75e665561e1bcb40403b5b',
+     x86_64: '1e52a3786e60afcc222911f6c143b9747238e6263f027030bd5862a8674ad99c',
   })
 
   def self.build

--- a/packages/openrtsp.rb
+++ b/packages/openrtsp.rb
@@ -8,8 +8,16 @@ class Openrtsp < Package
   source_sha256 '2db4f05616bdd21a609baf82c836486c44820c16a006315e02abe2b0b53a247e'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/openrtsp-2018.02.28-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/openrtsp-2018.02.28-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/openrtsp-2018.02.28-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/openrtsp-2018.02.28-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'a3e306b8e07bb82bd96414ca429f6ec395eb2edf1af39930920ef563b5c696d9',
+     armv7l: 'a3e306b8e07bb82bd96414ca429f6ec395eb2edf1af39930920ef563b5c696d9',
+       i686: 'ec6cffedbd6404b6175ddbdff1d64634c44a5c852d8232b94319c9cde6aab761',
+     x86_64: '8750845fd5e8469badc4a6d7c7c5026719efc788bbde3abfa1de083cb61732af',
   })
 
   def self.build

--- a/packages/packer.rb
+++ b/packages/packer.rb
@@ -21,8 +21,16 @@ class Packer < Package
   end
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/packer-1.2.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/packer-1.2.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/packer-1.2.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/packer-1.2.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '7d187c36dbede0977f43f6a7e70ffedffddfe39e3920453f8a399d19789295ec',
+     armv7l: '7d187c36dbede0977f43f6a7e70ffedffddfe39e3920453f8a399d19789295ec',
+       i686: '40fddfa96ac9bcb343ed0e5f4bf607a04268293dd42dc480457f59e78ab186b8',
+     x86_64: '7e9a790a93c8517ffd2a44c3cbca83d97bbfa5e6ac3c47cf37569c5d579fbdfc',
   })
 
   depends_on 'unzip' => :build

--- a/packages/packer_completion.rb
+++ b/packages/packer_completion.rb
@@ -8,8 +8,16 @@ class Packer_completion < Package
   source_sha256 '20ebfacd3f3a60f8dbd09e25b97d3b6e5049cbdf00a2d607fe79eaaef39e1eea'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/packer_completion-1.0.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/packer_completion-1.0.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/packer_completion-1.0.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/packer_completion-1.0.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '9334ea5ec25ee0207576ad8064e7d05bb3f7d56e38e788443992ee646f00eee7',
+     armv7l: '9334ea5ec25ee0207576ad8064e7d05bb3f7d56e38e788443992ee646f00eee7',
+       i686: 'ffe610cfa7cc113af4f2a6ec505aa2897aae771b0a3069b659fa53ae24aba902',
+     x86_64: '72db1d2343fc28baaaef56b8aaca87442b3d713f99ca3e3889b920a816f2c4b2',
   })
 
   def self.install

--- a/packages/percona_toolkit.rb
+++ b/packages/percona_toolkit.rb
@@ -8,8 +8,16 @@ class Percona_toolkit < Package
   source_sha256 '40f1a2241fb3ac1049e43ea75056f02d8af48c1eb731b48a4b564a054f8227e3'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/percona_toolkit-3.0.7-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/percona_toolkit-3.0.7-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/percona_toolkit-3.0.7-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/percona_toolkit-3.0.7-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'c805533b7b9ba184e17d070574c30c453a73a2dde3258787b50c6ac77e319495',
+     armv7l: 'c805533b7b9ba184e17d070574c30c453a73a2dde3258787b50c6ac77e319495',
+       i686: 'd71e086d9283cf638e461ec2d5917ddeeeda4ac0fb130a55cd86d436df5eb465',
+     x86_64: 'b3820dd0791160369b84a66017c6938ee6850074b9e3efefa2a1b4c64380727b',
   })
 
   depends_on 'percona_server'

--- a/packages/pthread_stubs.rb
+++ b/packages/pthread_stubs.rb
@@ -7,6 +7,19 @@ class Pthread_stubs < Package
   source_url 'https://xcb.freedesktop.org/dist/libpthread-stubs-0.3.tar.bz2'
   source_sha256 '35b6d54e3cc6f3ba28061da81af64b9a92b7b757319098172488a660e3d87299'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/pthread_stubs-0.3-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/pthread_stubs-0.3-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/pthread_stubs-0.3-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/pthread_stubs-0.3-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '12a930c63c5e155059b74c057b342ea4c94f5c1533dd27af55386cd72f920716',
+     armv7l: '12a930c63c5e155059b74c057b342ea4c94f5c1533dd27af55386cd72f920716',
+       i686: 'a832fa2df7eb69c355a13b2d910230c6dcfd1c49cb263cd787d225c6c751b5a6',
+     x86_64: '6d38609d0e19a07642e1e463f68101dffa20b505cb88a443497fbe985cc1e852',
+  })
+
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
     system "make"

